### DISCUSSION TITLE
Feature Request - drag and drop document contents to reorder

### DISCUSSION THREAD

**Discussion by accesstechnology-mike** [outline/outline#6921] [Ideas]
Word has this useful feature: content written with semantic sections (H1, H2, H3) can be moved to different document areas by drag and drop.

**Comment by sjwant** [comment]
You can move blocks of text by highlighting the text, holding down the fn (button on mac) and then dragging the text

**Comment by Mowd** [comment]
Highlight the text and hold the left click of a mouse for 1 sec, then you can drag the text block.

**Comment by EduNetEurope** [comment]
This seems to be a duplicate of https://github.com/outline/outline/discussions/1414
